docs/using-scylla/cassandra-compatibility.rst

```diff
@@ -63,6 +63,8 @@ Interfaces
 Supported Tools
 ---------------
 
+The tools are based on Apache Cassandra 3.11.
```

Comment:
According to 801f7848437a926169f263987970c94b5eeb87cf the accurate version is 3.11.13. But I guess that's not very important here, so ok.